### ISSUE TITLE
Support Python 3.7

### ISSUE THREAD

**Issue by arlyon** [netlify/build-image#228]
Similarly to #206, Python 3.7 was released in June. We're currently holding off on 3.7 features purely because we use sphinx powered documentation and which can't build on netlify without 3.7 available.

It would be nice to have, though I know `python3.7` and `python3.7-dev` are still in debian testing.

**Comment by bcomnes** [comment]
Working on it

**Comment by pradyunsg** [comment]
Hey @bcomnes! Any updates on this?

**Comment by bcomnes** [comment]
@pradyunsg

Yes actively developing a ubuntu 16.04 image.  We are also aiming to make the language version manager more flexible so you aren't limited to what its preinstalled.

**Comment by pradyunsg** [comment]
Glad to hear that! :)

**Comment by jayvdb** [comment]
It looks like Xenial images are now available, and I assume Python 3.7 work on those.  I found that Python 3.6 doesnt work on the new Xenial images -- not sure if that is expected behaviour.

**Comment by rminderhoud** [comment]
Project contains `runtime.txt` with a value of 3.7. Triggers a deploy failure:

```
10:56:39 PM: Starting build script
10:56:39 PM: Installing dependencies
10:56:39 PM: /usr/local/bin/run-build-functions.sh: line 168: /opt/buildhome/python3.7/bin/activate: No such file or directory
10:56:39 PM: failed during stage 'building site': Build script returned non-zero exit code: 1
10:56:39 PM: Error setting python version from runtime.txt
10:56:39 PM: Please see https://github.com/netlify/build-image/#included-software for current versions
10:56:39 PM: Error running command: Build script returned non-zero exit code: 1
```

**EDIT** Nevermind, seemed I was on an old build image since my site predated these changes. I had to manually go into the settings and change to 16.04 and it worked.

Deploy Settings -> Build Image Selection -> Edit Settings -> Ubuntu Xenial 16.04 (default)

**Comment by merwok** [comment]
Current version is even Python 3.8.3.

**Comment by arlyon** [comment]
@bcomnes Can this be closed or is there still work to be done?

**Comment by bcomnes** [comment]
Not sure, no longer personally contributing to this codebase.  My apologies.

**Comment by pradyunsg** [comment]
@arlyon: based on @rminderhoud's comment above, I think Python 3.7 is available as part of the Ubuntu 16.04 image (I haven't tested this myself)

> I had to manually go into the settings and change to 16.04 and it worked.
>
> Deploy Settings -> Build Image Selection -> Edit Settings -> Ubuntu Xenial 16.04 (default)

**Comment by arlyon** [comment]
Since this is possible (been using the image myself no problem) I'll close even if it's not the default and hopefully people will find this issue. Cheers all
